store/tikv/region_cache.go

```diff
@@ -166,14 +167,26 @@ func (r *RegionStore) follower(seed uint32, op *storeSelectorOp) AccessIndex {
 }
 
 // return next leader or follower store's index
-func (r *RegionStore) kvPeer(seed uint32, op *storeSelectorOp) AccessIndex {
+func (r *RegionStore) kvPeer(region *Region, seed uint32, op *storeSelectorOp) AccessIndex {
```

Comment:
Will we meet `learner` peer in this function?

Reply:
I think it's possible, ref #23272.